packages/grafana-ui/src/components/Table/styles.ts

```diff
@@ -51,7 +52,7 @@ export function useTableStyles(theme: GrafanaTheme2, cellHeightOption: TableCell
         width: overflowOnHover ? 'auto' : undefined,
         height: overflowOnHover ? 'auto' : `${rowHeight - 1}px`,
         minHeight: `${rowHeight - 1}px`,
-        wordBreak: overflowOnHover ? 'break-word' : undefined,
+        wordBreak: overflowOnHover && asCellText ? 'break-word' : undefined,
```

Comment:
`isLongString` or `asCellText`? is it a typo?

Reply:
Fixed, thanks! 🙏